### PR TITLE
Remove SDK Version callout

### PR DESCRIPTION
**Pull request by jonstelly**
Make it version-agnostic so it doesn't need changing in the future

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@jonstelly  that might work in this case. cc @scottaddie  
@spboyer do you agree?

File: aspnetcore/host-and-deploy/docker/building-net-docker-images.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The sample Dockerfile uses the [Docker multi-stage build feature](https://docs.d
 
 ## Prerequisites
 
-* [.NET Core 2.2 SDK](https://www.microsoft.com/net/core)
+* [.NET Core SDK](https://www.microsoft.com/net/core)
```
Comment (guardrex) [inline aspnetcore/host-and-deploy/docker/building-net-docker-images.md]: @Rick-Anderson The topic isn't versioned in metadata, but does it imply "2.2 or later"? If so, it should go ... and I think that's the wrong link. That link resolves to this :point_right: https://dotnet.microsoft.com/learn/dotnet/hello-world-tutorial/intro

```suggestion
* [.NET Core SDK 2.2 or later](https://dotnet.microsoft.com/download)
```

Comment (jonstelly) [reply aspnetcore/host-and-deploy/docker/building-net-docker-images.md]: I changed the target URL as suggested, that makes sense.

**Comment by Rick-Anderson** [comment]
@jonstelly  they need to know what version.

**Pull request merged**